### PR TITLE
docs: core docs migration (rst => mdx) - 6

### PR DESCRIPTION
**Pull request by rjitsu**
Child of #8048 

Files Migrated in this PR

<details>
<summary>docs/graphql/core/databases/bigquery/ ✅ </summary>

  - [x] docs/graphql/core/databases/bigquery/getting-started.rst ✅
  - [x] docs/graphql/core/databases/bigquery/index.rst ✅
  - [x] docs/graphql/core/databases/bigquery/schema/ ✅
      - [x] docs/graphql/core/databases/bigquery/schema/custom-field-names.rst ✅
      - [x] docs/graphql/core/databases/bigquery/schema/index.rst ✅
</details>

<details>
<summary>docs/graphql/core/getting-started/ ✅ </summary>

- [x] docs/graphql/core/getting-started/docker-simple.rst ✅
- [x] docs/graphql/core/getting-started/first-graphql-query.rst ✅
- [x] docs/graphql/core/getting-started/index.rst ✅
</details>

**NEW**

<details>
<summary>docs/graphql/core/databases/ ✅</summary>

- [x] docs/graphql/core/databases/connect-db.rst ✅
- [x] docs/graphql/core/databases/index.rst ✅
- [x] docs/graphql/core/databases/ms-sql-server/ ✅
- [x] docs/graphql/core/databases/ms-sql-server/mutations/ ✅
  - [x] docs/graphql/core/databases/ms-sql-server/mutations/delete.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/mutations/index.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/mutations/insert.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/mutations/upsert.rst ✅
- [x] docs/graphql/core/databases/ms-sql-server/queries/ ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/aggregation-queries.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/index.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/multiple-arguments.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/multiple-queries.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/nested-object-queries.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/pagination.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/performance.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/query-filters.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/simple-object-queries.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/sorting.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/queries/variables-aliases-fragments-directives.rst ✅
- [x] docs/graphql/core/databases/ms-sql-server/schema/ ✅
  - [x] docs/graphql/core/databases/ms-sql-server/schema/custom-field-names.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/schema/tables.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/schema/using-existing-database.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/schema/views.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/schema/table-relationships/ ✅
    - [x] docs/graphql/core/databases/ms-sql-server/schema/table-relationships/create.rst ✅
    - [x] docs/graphql/core/databases/ms-sql-server/schema/table-relationships/database-modelling.rst ✅
    - [x] docs/graphql/core/databases/ms-sql-server/schema/table-relationships/index.rst ✅
    - [x] docs/graphql/core/databases/ms-sql-server/schema/table-relationships/rename.rst ✅
- [x] docs/graphql/core/databases/ms-sql-server/subscriptions/ ✅
  - [x] docs/graphql/core/databases/ms-sql-server/subscriptions/index.rst ✅
  - [x] docs/graphql/core/databases/ms-sql-server/subscriptions/use-cases.rst ✅
- [x] docs/graphql/core/databases/postgres/schema/ ✅
  - [x] docs/graphql/core/databases/postgres/schema/default-values/ ✅
    - [x] docs/graphql/core/databases/postgres/schema/default-values/column-presets.rst ✅
    - [x] docs/graphql/core/databases/postgres/schema/default-values/created-updated-timestamps.rst ✅
    - [x] docs/graphql/core/databases/postgres/schema/default-values/postgres-defaults.rst ✅
    - [x] docs/graphql/core/databases/postgres/schema/default-values/sql-functions.rst ✅
  - [x] docs/graphql/core/databases/postgres/schema/remote-relationships/ ✅
    - [x] docs/graphql/core/databases/postgres/schema/remote-relationships/action-relationships.rst ✅
    - [x] docs/graphql/core/databases/postgres/schema/remote-relationships/remote-schema-relationships.rst ✅
    - [x] docs/graphql/core/databases/postgres/schema/remote-relationships/remote-source-relationships.rst ✅
 
</details>

---
🚧 - Work in Progress
⁉️ - Work Done - Review Pending
⚠️ - Review Done - Changes Required
✅ - Review Done - Migration Complete

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,3 +1,6 @@
 {
```
Comment (KRRISH96) [inline package-lock.json:1]: delete this file pls - its in root of repo (out of our work migration scope)

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 2]
```diff
@@ -62,25 +64,24 @@ Head to `http://localhost:8080/console` to open the Hasura console.
...
+  alt="Enter URL for existing database"
+  width="700px"
+/>
 
-</div>
-
-<div class="admonition">
+:::info Note
 
 Starting from scratch
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:86]: `Starting from scratch` is the correct title, pls fix and always cross check with live for content and UI correctness.
`:::info Starting from scratch`

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 3]
```diff
@@ -156,50 +159,48 @@ the link below. We send newsletters only once a month.
...
--   `Querying data <pg_queries>`: Use GraphQL queries to query data from
-    your GraphQL API.
--   `Inserting data <pg_mutations>`: Use GraphQL mutations to insert
-    data into your GraphQL API.
+- [Database modelling](/graphql/core/databases/postgres/schema/index.mdx#pg-schema): Learn how to model your database
+  schema, as well as how to extend it.
+- [Querying data](/graphql/core/databases/postgres/queries/index.mdx#pg-queries): Use GraphQL queries to query data from
+  your GraphQL API.
+- [Inserting data](/graphql/core/databases/postgres/mutations/index.mdx#pg-mutations): Use GraphQL mutations to insert
+  data into your GraphQL API.
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:167]: Pls remove hash-params for all 3

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 4]
```diff
@@ -156,50 +159,48 @@ the link below. We send newsletters only once a month.
...
-    used to execute custom business logic at specific points in time.
+- [Actions](/graphql/core/actions/index.mdx#actions): Actions can be used if you'd like to extend
+  your GraphQL schema by integrating with a REST endpoint.
+- [Remote schemas](/graphql/core/remote-schemas/index.mdx#remote-schemas): If you have an existing GraphQL
+  server or if you're comfortable with implementing one, you can use
+  remote schemas.
+- [Event triggers](/graphql/core/event-triggers/index.mdx#event-triggers): To trigger a serverless function
+  based on a database event, use event triggers.
+- [Scheduled triggers](/graphql/core/scheduled-triggers/index.mdx#scheduled-triggers): Scheduled triggers are
+  used to execute custom business logic at specific points in time.
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:182]: Please remove hash-params for all 4

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 5]
```diff
@@ -156,50 +159,48 @@ the link below. We send newsletters only once a month.
...
 
 ### Migrations
 
-Set up `Hasura migrations <migrations_setup>` to track your database
+Set up [Hasura migrations](/graphql/core/migrations/migrations-setup.mdx#migrations-setup) to track your database
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:186]: Pls remove hash-param

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 6]
```diff
@@ -156,50 +159,48 @@ the link below. We send newsletters only once a month.
...
-Docker, check out `deployment_docker`.
-
-<div class="admonition">
+Docker, check out [Run Hasura GraphQL engine using Docker](/graphql/core/deployment/deployment-guides/docker.mdx#deployment-docker).
 
-Additional Resources
+:::info Additional Resources
 
 Get Started with Hasura today - [Watch video
 guide](https://hasura.io/events/webinar/get-started-with-hasura/?pg=docs&plcmt=body&cta=getting-started&tech=).
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:204]: pls fix broken line break.

File: docs-new/docs/graphql/core/getting-started/docker-simple.mdx [hunk 7]
```diff
@@ -156,50 +159,48 @@ the link below. We send newsletters only once a month.
...
 endpoint and the Hasura console are not publicly accessible.
 
 ## Detailed Docker setup
 
 This was a quickstart guide to get the Hasura GraphQL engine up and
 running quickly. For more detailed instructions on deploying using
-Docker, check out `deployment_docker`.
-
-<div class="admonition">
+Docker, check out [Run Hasura GraphQL engine using Docker](/graphql/core/deployment/deployment-guides/docker.mdx#deployment-docker).
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/docker-simple.mdx:199]: Pls remove hash-param

File: docs-new/docs/graphql/core/getting-started/index.mdx [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+- Deploy the Hasura GraphQL engine, with access to a Postgres database
+  to store its metadata.
+- Connect new/existing database(s) and set up and test your GraphQL
+  API using the Hasura console UI _(also possible via CLI or API)_.
+- Consume the generated GraphQL API from your client apps.
+
+## Quickstart guides
+
+- [Using Hasura Cloud](/graphql/cloud/getting-started/index.mdx#cloud-getting-started) - Create a new Hasura
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/index.mdx:25]: please remove hash-param

File: docs-new/docs/graphql/core/getting-started/index.mdx [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+  Cloud project with just a few clicks.
+  - Recommended as it's **the fastest way** to try Hasura GraphQL
+    engine out.
+  - Comes with **extra features for reliability and security** and a
+    managed metadata database.
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/index.mdx:30]: pls fix line breaks

File: docs-new/docs/graphql/core/getting-started/index.mdx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+
+Hasura is open-source and can also be downloaded locally.
+[Set up Hasura GraphQL engine using Docker Compose](/graphql/core/getting-started/docker-simple.mdx#docker-simple).
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/index.mdx:33]: pls remove hash-param

File: docs-new/docs/graphql/core/getting-started/index.mdx [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+
+## Supported databases
+
+Hasura GraphQL engine supports:
+
+- [Postgres](/graphql/core/databases/postgres/index.mdx#database-postgres)
+- [MS SQL Server](/graphql/core/databases/ms-sql-server/index.mdx#database-ms-sql-server)
+- [Citus / Hyperscale](/graphql/core/databases/citus-hyperscale-postgres/index.mdx#database-citus-hyperscale-postgres)
+- [BigQuery](/graphql/core/databases/bigquery/index.mdx#database-bigquery)
+- **MySQL** (in preview) - See the [preview guide](/graphql/core/guides/mysql-preview.mdx#mysql-preview)
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/index.mdx:47]: pls remove hash-param for all 5 links

File: docs-new/docs/graphql/core/getting-started/index.mdx [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+
+:::info Additional Resources
+
+Get Started with Hasura today - [Watch video
+guide](https://hasura.io/events/webinar/get-started-with-hasura/?pg=docs&plcmt=body&cta=getting-started&tech=).
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/getting-started/index.mdx:52]: pls fix line break

**Review by KRRISH96 — CHANGES_REQUESTED** [review]

File: docs-new/docs/graphql/core/getting-started/index.mdx (hunk 9 above)
Comment (rjitsu) [reply docs-new/docs/graphql/core/getting-started/index.mdx:30]: I think this is supposed to be nested, I checked the previous docs, can you please confirm?

Comment (KRRISH96) [reply docs-new/docs/graphql/core/getting-started/index.mdx:30]: This is nested. I was referring to the content text itself. Only 1 or two words in second line even if first line is super short.

```suggestion
  - Recommended as it's **the fastest way** to try Hasura GraphQL engine out.
  - Comes with **extra features for reliability and security** and a managed metadata database.
```

File: docs-new/docs/graphql/core/databases/ms-sql-server/subscriptions/use-cases.mdx [hunk 13]
```diff
@@ -210,14 +209,15 @@ This view will have the following fields: `poll_id`, `option_id` and
 `votes`, i.e. it gives the number of votes received by each option for a
 poll.
 
-Next, `set up relationships <ms_sql_server_table_relationships>` `poll`
+Next, [set up relationships](/graphql/core/databases/ms-sql-server/schema/table-relationships/index.mdx) `poll`
 and `option` between the `poll_results` view and the `poll` and `option`
 tables using the `poll_id` and `option_id` fields respectively.
 
 Now we can use the following subscription to display the latest poll
 result:
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/databases/ms-sql-server/subscriptions/use-cases.mdx:217]: Please fix line break

File: docs-new/docs/graphql/core/databases/ms-sql-server/schema/views.mdx [hunk 14]
```diff
@@ -23,34 +26,22 @@ Hasura GraphQL engine lets you expose views over the GraphQL API to
...
 
--   Head to the `Data -> SQL` section of the Hasura console
--   Enter your [create view SQL
-    statement](https://docs.microsoft.com/en-us/sql/relational-databases/views/create-views?view=sql-server-ver15)
--   Hit the `Run` button
-
-</div>
+- Head to the `Data -> SQL` section of the Hasura console
+- Enter your [create view SQL
+  statement](https://docs.microsoft.com/en-us/sql/relational-databases/views/create-views?view=sql-server-ver15)
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/databases/ms-sql-server/schema/views.mdx:38]: pls fix line break

File: docs-new/docs/graphql/core/databases/postgres/schema/default-values/column-presets.mdx [hunk 15]
```diff
@@ -1,83 +1,72 @@
...
+with a particular [user role](/graphql/core/auth/authorization/roles-variables.mdx).
 
 Hasura GraphQL engine's column presets let you define role-based values
 for any field/column. These values can either be a session variable
 value or a static value.
 
-<div class="admonition">
-
-Column preset restricts mutation access for configured role
+:::info Column preset restricts mutation access for configured role
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/databases/postgres/schema/default-values/column-presets.mdx:30]: Lets use `:::caution`

File: docs-new/docs/graphql/core/databases/postgres/schema/remote-relationships/action-relationships.mdx [hunk 16]
```diff
@@ -0,0 +1,218 @@
...
+
+```bash
+hasura metadata apply
+```
+
+</TabItem>
+<TabItem value="api" label="API">
+
+You can create an action relationship when defining custom types via the
+`metadata_set_custom_types` metadata API:
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/databases/postgres/schema/remote-relationships/action-relationships.mdx:106]: missed link

File: docs-new/docs/graphql/core/databases/ms-sql-server/queries/query-filters.mdx [hunk 17]
```diff
@@ -57,7 +62,7 @@ Here `_eq` and `_gt` are examples of comparison operators that can be
 used in the `where` argument to filter on equality.
 
 You can see the complete specification of the `where` argument in the
-`API reference <WhereExp>`.
+[API reference](/graphql/core/api-reference/graphql-api/query.mdx).
```
Comment (KRRISH96) [inline docs-new/docs/graphql/core/databases/ms-sql-server/queries/query-filters.mdx:65]: link should take to a subsection - please add hash param (`#whereexp`)

**Review by KRRISH96 — COMMENTED** [review]
4 tiny changes and we are GTG

1. Missing position for `docs-new/docs/graphql/core/databases/ms-sql-server/schema/default-values/_category_.json`
2. Commented below
3. and 4. Unresolved comments above

**Review by KRRISH96 — APPROVED** [review]

**Pull request merged**